smart_open/tests/test_asb.py

```diff
@@ -324,7 +329,11 @@ def test_iter(self):
         put_to_container(blob_name, contents=expected)
 
         # connect to fake Azure Storage Blob and read from the fake key we filled above
-        fin = smart_open.asb.Reader(CONTAINER_NAME, blob_name, client=test_blob_service_client)
+        fin = smart_open.asb.Reader(
+            CONTAINER_NAME,
+            blob_name,
+            client=test_blob_service_client
+        )
```

Comment:
-1: why?

Reply:
What do you mean with this comment?

Reply:
I think @piskvorky thinks that your change to that particular line is a bad idea, and is asking for the motivation behind it.

Why are you breaking a short-ish line into three very short lines?

Reply:
> The CI builds are failing. Can you please make them pass?
> 
> https://travis-ci.org/github/RaRe-Technologies/smart_open/builds/673755778

The CI builds are now passing

Reply:
> It appears @piskvorky thinks that your change to that particular line is a bad idea, and is asking for the motivation behind it.
> 
> Why are you breaking a short-ish line into three very short lines?

Ok I will change this, I had to do it for long lines and so did for all occurrences, I will undo this for short lines